genie-common-internal/src/main/java/com/netflix/genie/common/internal/dto/v4/converters/JobSpecificationServiceConverter.java

```diff
@@ -88,62 +93,86 @@ public static GetJobSpecificationRequest toProtoGetJobSpecificationRequest(final
         return GetJobSpecificationRequest.newBuilder().setId(id).build();
     }
 
+//    /**
```

Comment:
Meant to add all this commented out code?

Reply:
Yeah I left a bunch of this stuff in due to expecting to need it once the reservation of ID code is written it's going to be moved there. in there interest of getting this done quickly I left it like this